Integrations/integration-Kenna.yml

```diff
@@ -0,0 +1,487 @@
+                Type: entryTypes.note,
+                Contents: res,
+                ContentsFormat: formats.json,
+                ReadableContentsFormat : formats.markdown,
+                EntryContext : context,
+                HumanReadable: tableToMarkdown('Kenna Vulnerabilities', hr)
+            };
+    }
+
+    function searchVulnerabilitiesRequest(urlQuery) {
+        var requestUrl = url.replace(/[\/]+$/, '') + '/vulnerabilities/search' + encodeToURLQuery(urlQuery);
+        var res = http(
+            requestUrl,
+            {
+                Method: 'GET',
+                Headers: {
+                    'X-Risk-Token': [apiKey],
+                    'Content-Type': ['application/json'],
+                    'Accept': ['application/json']
+                }
+            },
+            params.insecure,
+            params.proxy
+        );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(res.Body);
+    }
+
+
+    function getConnectors() {
+        var response = getConnectorsRequest();
+        var res = response.connectors;
+        var hr = [];
+        var context = {};
+        if (!context.Kenna) {
+            context.Kenna = {};
+        }
+        context.Kenna.ConnectorsList = []
+        for (var i = 0; i < res.length; i++) {
+            hr[i] = {
+                'Host' : res[i].host,
+                'Name' : res[i].name,
+                'Running' : res[i].running,
+                'ID' : res[i].id
+            };
+            context.Kenna.ConnectorsList[i] = {};
+            context.Kenna.ConnectorsList[i].ID = res[i].id;
+        }
+        return {
+            Type: entryTypes.note,
+            Contents: res,
+            ContentsFormat: formats.json,
+            ReadableContentsFormat : formats.markdown,
+            EntryContext : context,
+            HumanReadable: tableToMarkdown('Kenna', hr)
+        };
+    }
+
+    function getConnectorsRequest() {
+        var requestUrl = url.replace(/[\/]+$/, '') + '/connectors';
+        var res = http(
+            requestUrl,
+            {
+                Method: 'GET',
+                Headers: {
+                    'X-Risk-Token': [apiKey],
+                    'Content-Type': ['application/json'],
+                    'Accept': ['application/json']
+                }
+            },
+            params.insecure,
+            params.proxy
+        );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(res.Body);
+    }
+
+    function runConnector(id) {
+        try {
+                var response = runConnectorRequest(id);
+                if (response.success === 'true') {
+                    return 'Connector ran successfully!';
```

Comment:
ran => run

Reply:
It actually should be in past tense, as the connection started running once then command was sent.

Reply:
ok